completions/apt-mark

```diff
@@ -0,0 +1,52 @@
+# Debian apt-mark(8) completion                             -*- shell-script -*-
+
+_apt_mark()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+
+    local special i
+    for (( i=0; i < ${#words[@]}-1; i++ )); do
+        if [[ ${words[i]} == @(hold|unhold|auto|manual|install|remove|purge|showhold|showauto|showmanual|showinstall|showremove|showpurge|minimize-manual) ]]; then
+            special=${words[i]}
```

Comment:
Should we `break` after this?

Reply:
Actually, since using more than one special option results in an error, here's a potential solution using an array, effectively disabling the completion if the user inputs more than one option:
```diff
diff --git a/completions/apt-mark b/completions/apt-mark
index a88aeacd..213ab6a4 100644
--- a/completions/apt-mark
+++ b/completions/apt-mark
@@ -5,15 +5,19 @@ _apt_mark()
     local cur prev words cword split
     _init_completion -s || return
 
-    local special i
+    local i
+    local special=()
     for (( i=1; i < ${#words[@]}-1; i++ )); do
         if [[ ${words[i]} == @(auto|manual|minimize-manual|showauto|showmanual|hold|unhold|showhold|install|remove|purge|showinstall|showremove|showpurge) ]]; then
-            special=${words[i]}
+            special+=(${words[i]})
         fi
     done
+    if [[ ${#special[@]} -gt 1 ]]; then
+        return
+    fi
 
-    if [[ -n $special ]]; then
-        case $special in
+    if [[ -n ${special[0]} ]]; then
+        case ${special[0]} in
             minimize-manual)
                 return
                 ;;
```

Reply:
> Actually, since using more than one special option results in an error,

True, but:
```
# apt-mark unhold hold
E: Unable to locate package hold
E: No packages found
```
So the two specials are not actually treated as specials, the latter is treated as a package name. It just happens I don't have a package named "hold" installed, that's why the error. I don't think we should prevent cases like that from happening, so I opted for the simple `break`.